packages/ember/tests/routing/basic_test.js

```diff
@@ -321,90 +321,6 @@ QUnit.test('Model passed via renderTemplate model is set as controller\'s model'
   equal(jQuery('p:contains(emberjs)', '#qunit-fixture').length, 1, 'Passed model was set as controllers model');
 });
 
-test('Renders correct view with slash notation', function() {
-  setTemplate('home/page', compile('<p>{{view.name}}</p>'));
-
-  Router.map(function() {
-    this.route('home', { path: '/' });
-  });
-
-  App.HomeRoute = Route.extend({
-    renderTemplate() {
-      this.render('home/page');
-    }
-  });
-
-  App.HomePageView = EmberView.extend({
-    name: 'Home/Page'
-  });
-
-  bootApplication();
-
-  equal(jQuery('p:contains(Home/Page)', '#qunit-fixture').length, 1, 'The homepage template was rendered');
-});
-
-test('Renders the view given in the view option', function() {
```

Comment:
What I'm saying is the API was never deprecated for 2.4 so I'm not sure if can be removed. This is a question for someone on core.